static/css/yui-guide.css

```
@@ -62,32 +57,146 @@ body.yui-guide-home-driver-hidden .driver-highlight {
+      rgba(186, 231, 255, 0.98) 13%,
+      rgba(76, 137, 255, 0.95) 52%,
+      rgba(39, 89, 228, 0.98) 96%,
+      transparent 100%
+    ) bottom right / var(--yui-guide-spotlight-corner-size) var(--yui-guide-spotlight-corner-size) no-repeat,
+    radial-gradient(circle at bottom left,
+      rgba(235, 249, 255, 0.98) 0,
+      rgba(186, 231, 255, 0.98) 13%,
+      rgba(76, 137, 255, 0.95) 52%,
+      rgba(39, 89, 228, 0.98) 96%,
+      transparent 100%
+    ) bottom left / var(--yui-guide-spotlight-corner-size) var(--yui-guide-spotlight-corner-size) no-repeat;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)